src/search.cpp

```diff
@@ -99,7 +99,7 @@ void update_correction_history(const Position& pos,
     const Move  m  = (ss - 1)->currentMove;
     const Color us = pos.side_to_move();
 
-    const int nonPawnWeight = 165;
+    static constexpr int nonPawnWeight = 165;
```

Comment:
what does static actually do for a single constant like this?

Reply:
Resolved